src/hooks/useConnectSovereignVenue.ts

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from "react";
+
+import { AnyVenue } from "types/venues";
+
+import { useSelector } from "./useSelector";
+import { useFirestoreConnect } from "./useFirestoreConnect";
+import { useSovereignVenueId } from "./useSovereignVenueId";
+import {
+  isSovereignVenueRequestedSelector,
+  sovereignVenueSelector,
+} from "utils/selectors";
+import { WithId, withId } from "utils/id";
```

Comment:
I think we should reorder the imports here. 

```suggestion
import { WithId, withId } from "utils/id";
import {
  isSovereignVenueRequestedSelector,
  sovereignVenueSelector,
} from "utils/selectors";

import { useSelector } from "./useSelector";
import { useFirestoreConnect } from "./useFirestoreConnect";
import { useSovereignVenueId } from "./useSovereignVenueId";
```

Reply:
Changed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0c1f9f07a5696848014d78ad2604de3c1ed2d194